### DISCUSSION TITLE
larry77 I'm confused what did you mean by the chapter thing?

### DISCUSSION THREAD

**Discussion by DrewThomasson** [DrewThomasson/VoxNovel#7] [General]
where you said "Let us say that the first book generated 11 chapter text files and the
second one only 5. Then 6 extra chapters were added to the generated audiobook."

**Comment by larry77** [comment]
I meant that the chapters 6 to 11 were not cancelled, so they were finally added to the audiobook, which consisted of the new book plus part of the old book.

**Comment by DrewThomasson** [comment]
-Updated English sentence splitting with a improved version, No more tts saying lists and such awkwardly with a bunch of pauses!
 -The new recursive method worked, sentences are as long as possible without running into any errors so far!

-Oh btw if you have any future feature suggestions would love to hear no matter how ridiculous, like for instance a ridiculous one form me is that I'm planning on adding auto generated sound effects somehow lol.(For like when a book describes a location it'll play the sound effect for it.)

**Reply by larry77** [reply]
That is actually something I had in mind. For instance, if the book talks about a thunderstorm, it would be nice to have the sound of thunder in the background ;-) .
